apps/vh/notification-api/prod.yaml

```diff
@@ -8,12 +8,10 @@ spec:
   values:
     java:
       ingressHost: vh-notification-api.platform.hmcts.net
+      environment:
+        SERVICES__NOTIFICATIONAPIRESOURCEID: http://vh-notification-api.platform.hmcts.net
+        NotifyConfiguration__Environment: Prod
```

Comment:
could this be in the notification api chart instead with the value `{{ .Values.global.environment }}` which will replace it wiht the environment